### ISSUE TITLE
PANTHER:PTN000042666

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#2174]
and again, I don't think I'll ever eradicate this one

SPAC11E3.12 | mitochondrial thioredoxin family protein, implicated in sulfur cluster assembly |   | mitochondrial electron transport, NADH to ubiquinone |   | GO_Central | Schizosaccharomyces pombe | IBA | PANTHER:PTN000042666UniProtKB:P19404 | nadh dehydrogenase ubiquinone flavoprotein 2, mitochondrial pthr10371 | protein |   | PMID:21873635 | 20170228

  | SPAC11E3.12 | mitochondrial thioredoxin family protein, implicated in sulfur cluster assembly |   | mitochondrial respiratory chain complex I |   | GO_Central | Schizosaccharomyces pombe | IBA | PANTHER:PTN000042666TAIR:locus:2132387 more... | nadh dehydrogenase ubiquinone flavoprotein 2, mitochondrial pthr10371 | protein |   | PMID:21873635 | 20170228

  | SPAC11E3.12 | mitochondrial thioredoxin family protein, implicated in sulfur cluster assembly |   | NADH dehydrogenase (ubiquinone) activity |   | GO_Central | Schizosaccharomyces pombe | IBA | PANTHER:PTN000042735UniProtKB:P19404 | nadh dehydrogenase ubiquinone flavoprotein 2, mitochondrial pthr10371 | protein

**Comment by marcfeuermann** [comment]
ID; PTHR10371
I understand your frustration Val, Anyway, this one has been addressed and corrected about a month ago. So it should disappear very soon.

**Comment by ValWood** [comment]
;) You are on fire today! Are you going for a ticket closing record?

**Comment by ValWood** [comment]
So far, I have checked PAINT annotations for "Unknowns- those I thought should not have a process annotation, and I've checked those which were annotated to a GO slim term by PAINT, which we/PomBase had not annotated to the same slim term. That should capture most of the larger errors, at least for  conserved cell level processes. 
Some time in the spring we will import the PAINT annotations into PomBase. At this point I will probably find some more issues, but I think they will  be minor. ...After that component, and function.... I only think about process right now... but not so many more I hope!
